hbase-server/src/main/java/org/apache/hadoop/hbase/replication/ReplicationSinkServiceImpl.java

```diff
@@ -75,7 +75,8 @@ public void initialize(Server server, FileSystem fs, Path logdir, Path oldLogDir
   public void startReplicationService() throws IOException {
     this.replicationSink = new ReplicationSink(this.conf);
     this.server.getChoreService().scheduleChore(
-      new ReplicationStatisticsChore("ReplicationSinkStatistics", server, statsPeriod));
+        new ReplicationStatisticsChore("ReplicationSinkStatistics", server,
+            1000 * statsPeriodInSecond));
```

Comment:
You can use `TimeUnit.SECONDS.toMillis(statsPeriodInSecond)`

Reply:
OK, thanks for reviewing.